proto/substrait/algebra.proto

```diff
@@ -369,6 +369,8 @@ message Rel {
     //Physical relations
     HashJoinRel hash_join = 13;
     MergeJoinRel merge_join = 14;
+
+    DdlRel ddl = 15;
```

Comment:
This is a new category of relations so it probably requires a comment defining that category, perhaps "write relations".

Reply:
added "write relations" comment
in addition WriteRel object is also added
because if we add the DdlRel there is no reason not to add also WriteRel